Description:Remove gemspec git ls files.patch
 Removed call to 'git ls-files' from gem spec.
 Use also a require without path to load the version number
Subject:Remove gemspec git ls-files
From:Per Andersson <avtobiff@gmail.com>
---
 pygments.rb.gemspec |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/pygments.rb.gemspec
+++ b/pygments.rb.gemspec
@@ -1,4 +1,8 @@
-require File.expand_path('../lib/pygments/version', __FILE__)
+version_file = File.expand_path('../lib/pygments/version.rb', __FILE__)
+if not File.exists?(version_file)
+  version_file = 'pygments/version'
+end
+require version_file
 
 Gem::Specification.new do |s|
   s.name = 'pygments.rb'
@@ -21,5 +25,5 @@
   # s.extensions = ['ext/extconf.rb']
   s.require_paths = ['lib']
 
-  s.files = `git ls-files`.split("\n")
+#  s.files = `git ls-files`.split("\n")
 end
